packages/experiments/src/components/Text/Text.tsx

```diff
@@ -1,7 +1,11 @@
 import * as React from 'react';
-import { IStyle } from '../../Styling';
-import { createComponent, IStyleProps, IViewProps } from './createComponent';
-import { IFontTypes, IFontFamilies, IFontSizes, IFontWeights, IFontColors } from './theming/ITypography';
+import { IStyle, ITheme, IFontStyles, IPalette } from '../../Styling';
+import { TViewProps, createComponent } from '../../utilities/createComponent';
+import { IFontTypes, IFontFamilies, IFontSizes, IFontWeights } from './theming/ITypography';
+
+export type IStyleProps<TProps, TStyles> = TProps & {
+  theme: ITheme;
+};
```

Comment:
This should be provided by foundation (it is in #5441) so you can move this to createComponent or leave it marked with a TODO to remove. Thanks